sdk/storage/Azure.Storage.DataMovement/src/JobPartInternal.cs

```diff
@@ -20,12 +20,12 @@ internal abstract class JobPartInternal
         public QueueChunkDelegate QueueChunk { get; internal set; }
```

Comment:
I realized this was internal so I stopped -- but leaving it here just to show what my changes look like.